lib/iris/fileformats/pp.py

```diff
@@ -221,31 +221,6 @@
                        'default': np.dtype('>f4'),
                        }
 
-# LBPROC codes and their English equivalents
```

Comment:
I think moving these to iris.fileformats.rules is a mistake, as it the information is specific to PP (and FF), whereas iris.fileformats.rules "ought" to be the home for fileformat-independent rules processing.
They aren't specially tied to implementation of text rules anyway.

In any case, "LBPROC_PAIRS" and "lbproc_map" are perfectly reasonable public elements of the API, so you ***can't just move them*** anyway.

In an *ideal* world I think these would be in "iris.fileformats.um_cf_map", and also "lbroc_map" should be capitalised, but it's a bit late for that...

Reply:
(from discussion with @dkillick)
The move was introduced because of a circularity problem when `iris.fileformats.pp` imports `iris.fileformats.pp_save_rules`, which imports `iris.fileformats.pp` ...
By putting 'lbproc_map' into `rules`, then `pp_save_rules` no longer needs to import `pp`.

New suggestion: Put LBPROC_PAIRS and `lbproc_map` inside pp_save_rules itself.
Then to avoid breaking existing API, publish them in 'pp' by importing them from 'pp_save_rules'.

Reply:
More trouble...
I missed that "lbproc_map" is also used in "pp_rules" 
( i.e. "pp_load_rules", perhaps it should now be called that ? )

You can import it there from "pp_save_rules", but that is obviously a rather peculiar dependency ordering.
You cannot import it from 'pp', if 'pp' is to import 'pp_rules' (same circularity problem as before)

In my view, this info ***really*** belongs with the other similar stuff in 'um_cf_map'.
To do this *properly*, we should really put this info into metarelate, and modify`tools/gen_translations.py` to support it, but I really don't think we have time for that.
In which case, we should just host this translation info in a separate module.
See : https://github.com/dkillick/iris/pull/12 for worked suggestion ...

Reply:
> i.e. "pp_load_rules", perhaps it should now be called that ? 

See #2790.

>  we should just host this translation info in a separate module.

Agreed 👍 Thanks for spotting this, @pp-mo